django/pyproject.toml

```diff
@@ -63,6 +63,7 @@ flake8-pytest-style = "^1.3.0"
 pep8-naming = "^0.11.1"
 watchdog = {extras = ["watchmedo"], version = "^1.0.2"}
 mypy-boto3-s3 = "^1.17.47"
+pytest-freezegun = "^0.4.2"
```

Comment:
It does freeze the time during the test as well, making it deterministic. There's a few other tests it could be applied to as well, so I figured leaving it is alright.